src/main/java/core/figures/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,31 @@
+package core.figures;
+
+import java.text.DecimalFormat;
+import randomizers.sourcerand.Colors;
+
+public class IsoscelesTrapezoid extends Figure {
+
```

Comment:
let's remove this empty line) and look through all classes and remove empty lines like this)